CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.4)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Do we really need that?

Reply:
Yes. This is the line which makes the magic happen:

with VERSION 2.4:
```
-- Could NOT find OpenMP_C (missing: OpenMP_C_FLAGS OpenMP_C_LIB_NAMES) 
-- Could NOT find OpenMP_CXX (missing: OpenMP_CXX_FLAGS OpenMP_CXX_LIB_NAMES) 
-- Could NOT find OpenMP (missing: OpenMP_C_FOUND OpenMP_CXX_FOUND) 
```
with VERSION 3.12:
```
-- Found OpenMP_C: -Xclang -fopenmp (found version "3.1") 
-- Found OpenMP_CXX: -Xclang -fopenmp (found version "3.1") 
-- Found OpenMP: TRUE (found version "3.1")
```

(This is from two runs on Catalina. One after the other with a clean build directory, the only difference being the line you asked about.)

I'm not sure anymore where I found the solution, but it's mentioned in a stackoverflow answer here:
> For others seeing this in the future, update to CMake 3.12 if it has been released at the time of reading.

https://stackoverflow.com/questions/49738961/cmake-support-for-openmp-on-macos-high-sierra/50313962

Reply:
Ok, thanks!